### DISCUSSION TITLE
Homebrew formula

### DISCUSSION THREAD

**Discussion by calebeaires** [PostgREST/postgrest#2073] [Q&A] [answered]
postgrest 9 has came to life! It has great new feature. Do you folks has some plain to update [Homebrew formula](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/postgrest.rb) to this new version? 

Congrats!

**Comment by steve-chavez** [comment]
@calebeaires Could you help us with that :pray: ? We've not been modifying the Homebrew formula.

**Reply by calebeaires** [reply]
I tried but I had no success. After testing the postgrest does not run quite well, maybe because the M1 Apple Silicon..

**Comment by wolfgangwalther** [comment, marked as answer]
Seems like the homebrew formula is at v10 by now, so it's getting updated.
